### ISSUE TITLE
tractor with seeder makes mysterious turns at end of row - Racetrack pattern

### ISSUE THREAD

**Issue by geo-luchs** [Courseplay/courseplay#4545]
Hello

I've created CP course with mode
"circle form".
 
When using a tractor with seeder , e.g. Pronto 9 DC, he is making crazy turn overs.

He is not going directly in the correct direction, but always in the other direction.
in screenshot I marked the current way red, but I would expect the green way.

Green radius is larger than he is driving the red radius. So it should be possible to use the green way.

![mystirous_turn](https://user-images.githubusercontent.com/53658780/68535689-9a53d200-0346-11ea-85a5-3fc088a82259.png)

Int this screenshot green way is to the left, but he turns first to the right.
Otherwise if he need to turn to the right, he is turning first to the left.

**Comment by Tensuko** [comment]
logfile wich channel 12 and 14 maybe helps

**Comment by pvaiko** [comment]
And a full log.

**Comment by geo-luchs** [comment]
Hi

I did now a test with 12 +14
[log.txt](https://github.com/Courseplay/courseplay/files/3828482/log.txt)

It is a very easy turn, but he still do this crazy turn:
![Turn_1](https://user-images.githubusercontent.com/53658780/68544626-6a97df00-03c5-11ea-8db9-50d7ad41f0a4.png)
![turn_2](https://user-images.githubusercontent.com/53658780/68544629-6ec3fc80-03c5-11ea-8b0d-51dcbafe6d2e.png)

**Comment by Tensuko** [comment]
Question: how do you create the screenshots and why you cut them so weird ?
Just press the print key on your keyboard and in the log you will find the position, then post them.
Can't see the full turn and not even the waypoints of your course.
The more Information you provide, the more we can help you.

**Comment by geo-luchs** [comment]
I made screenshots with Fast stone capture before.

Now without Fast stone capture, only windows print.

is that better?
![grafik](https://user-images.githubusercontent.com/53658780/68544790-0e35bf00-03c7-11ea-93bf-a26dce903f2e.png)
![grafik](https://user-images.githubusercontent.com/53658780/68544844-7be1eb00-03c7-11ea-9150-9fdbe6932147.png)
![grafik](https://user-images.githubusercontent.com/53658780/68544852-8308f900-03c7-11ea-9626-d88f760bb795.png)
![grafik](https://user-images.githubusercontent.com/53658780/68544861-87cdad00-03c7-11ea-93cc-e9466f13a0ee.png)
![grafik](https://user-images.githubusercontent.com/53658780/68544865-8ef4bb00-03c7-11ea-80a8-9867142fc774.png)

**Comment by pvaiko** [comment]
Yeah, that's a tricky one. In the meanwhile, try with the AI Turns enabled, that should work fine.

**Comment by Tensuko** [comment]
@geo-luchs 
yes, that is nicer to see whats happening there, course, start, target and stuff. A bit more overview.
As @pvaiko said, use AITurn enable under Advanced Settings.

**Comment by geo-luchs** [comment]
Unfortunately, even when using AITurn enabled, it makes the same turn. nothing changed

**Comment by pvaiko** [comment]
It would work if you had the latest version (334) but you have 333. If it does not work with the latest release and AI Turns on then post a full log with 12 and 14.

**Comment by geo-luchs** [comment]
puuuh, it's hard to follow your versions ;-)
I believe, I've downloaded 333 yesterday evening ;-)

OK, I will download it. Thanks for your support.

**Comment by pvaiko** [comment]
closing due to inactivity
